tests/Operation/DeleteTest.php

```diff
@@ -1,11 +1,12 @@
 <?php
 
+declare(strict_types=1);
```

Comment:
I assume this is the same reason as `ChangeStreamIteratorTest.php`, so this may warrant making a concise comment that you can copy-paste between files.